docs/console/tutorials/create-extension.md

```diff
@@ -4,204 +4,267 @@ title: Getting started
-}
-```
+Remember to save your changes by clicking the _Apply_ button from the _Advanced_ tab when you finish these updates, then to finalize the configuration of the Composer page with the _Publish all changes_ button at the top-right side of the page.
 
-You can find out available capabilities in the [Identity and access management page](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#identity-capabilities-inside-console).
-:::
+### 5. Call the extension
 
-:::info
-These registered routes are rendered as a menu item with label `Integrated Backoffice` that is attached to a `category` menu group with label `My menu group`. If you do not want to create new menu group, you can 
-- define the `category.id` value using an [existing menu groups](/console/console-extensibility/locations.md) corresponding to the chosen location applied as `destination.id` (e.g. `runtime` for the project location). In this case the `category.labelIntl` and `category.order` should be undefined.
-- not define the `category` field, so that the menu item will not be attached on any menu group 
-:::
+After every update, the extension is automatically updated.
+After updating the Composer page, we can finally move to the Extension and see the table with all data coming from Jira.
 
+![The extension shows the table with the Jira information](./img/extensions-final-result.png)
 
-**Response on success**
-```markdown
-Successfully applied extension with id `my-extension-id`
-```
+## Additional guide: manage extensions with miactl
+
+In this section, we will explain how to create, review and activate extensions using [miactl](/cli/miactl/10_overview.md).
+
+Remember to install the CLI if you haven't done it yet by following the instructions in the [CLI installation](/cli/miactl/20_setup.md)section and
+to [set up the context](/cli/miactl/30_commands.md#context) with the Company ID where you want to create the extension.
 
-### Edit the registered extension
+### Create and activate the extension
 
-If you need to correct or modify your newly registered extension, you can do so using the same miactl command and specifing the extension ID with the proper flag:
+You can also replicate the tutorial by creating the extension via _miactl_. In your CLI type the following command:
 
 ```sh
-miactl extensions apply --endpoint https://<my-console-domain>/ --company-id my-tenant-id -f ./<relative-path>/my-extension.json --extension-id my-extension-id 
+miactl extensions apply --endpoint https://<my-console-domain>/ --company-id my-tenant-id -f ./<relative-path>/my-extension.json
 ```
 
-<details>
-  <summary>See the complete example</summary>
-
-The `my-extension.json` file should always contains all the extension info, both those to edit and both those that are already registered:
+where `my-tenant-id` is the identifier of the company, and the `my-extension.json` is a JSON file includes the following content:
```

Comment:
Do we have a link where we redirect to explain the individual fields?

Reply:
Unfortunately, we do not have it yet. We should consider improving the miactl documentation with the schema of each item that can be included (extensions, applications, etc).